JS_Functions/task2.js

```diff
@@ -0,0 +1,8 @@
+const calculateAge = (age) => {
```

Comment:
I would suppose the function with name calculateAge should return number

Reply:
you can name it like `checkIfAdult` or simply `isAdult` (it is exception for function which return boolean, sometimes it can be named like boolean variable)